### PR TITLE
release(v0.3.0): add release notes and release announcement

### PR DESCRIPTION
**Pull request by Xunzhuo**
Signed-off-by: bitliu <bitliu@tencent.com>

### CODE REVIEW

File: docs/latest/releases/v0.3.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+skip_list: true
+aliases:
+- /releases/v0.3
+- /releases/v0.3.0
+---
+# Envoy Gateway Release v0.3
+
+We are pleased to announce the release of Envoy Gateway v0.3!
+
+This is the first functional release of Envoy Gateway. We would like to thank the entire Envoy Gateway community for
```
Comment (arkodg) [inline docs/latest/releases/v0.3.md:17]: first functional ?

File: release-notes/v0.3.0.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      Added Support for the UDPRoute API
+      Added Support for the GRPCRoute API
+      Added Support for HTTPRoute URLRewrite Filter
+      Added Support for HTTPRoute RequestMirror Filter
+      Added Support for HTTPRoute ResponseHeaderModifier Filter
+      Added Support for Request Authentication 
+      Added Support for Global Rate Limiting
+      Added Support for Routes ReferenceGrant
+      Added Support for Namespace Server Config Type
+      Added Support for Managing EnvoyProxy Deployment
```
Comment (danehans) [inline release-notes/v0.3.0.yaml:35]: s/Added Support for Managing EnvoyProxy Deployment/Added initial management of Envoy Proxy deployment via EnvoyProxy API/

**Review by danehans — CHANGES_REQUESTED** [review]

File: docs/latest/releases/v0.3.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: Announcing Envoy Gateway v0.3
+linktitle: v0.3
+subtitle: Major Update
+description: Envoy Gateway v0.3 release announcement.
+publishdate: 2022-10-20
```
Comment (danehans) [inline docs/latest/releases/v0.3.md:6]: Update this date.

**Comment by danehans** [comment]
@Xunzhuo CI failed due to https://github.com/envoyproxy/gateway/issues/1016.

**Comment by danehans** [comment]
@Xunzhuo please note https://github.com/envoyproxy/gateway/pull/1021 and update the release notes accordingly.

**Comment by danehans** [comment]
Waiting for https://github.com/envoyproxy/gateway/pull/1021 to merge and will then rerun CI.

**Comment by danehans** [comment]
@Xunzhuo xan you rebase to consume https://github.com/envoyproxy/gateway/commit/d016a662876abad67a6481f6c334f05bd15a262b that resolves the CI issues?

File: release-notes/v0.3.0.yaml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    change: |
+      Fixed Make Image Failed in Darwin
+      Fixed Wait for Job Succeeded before conformance test
+      Upgraded Echoserver Image Tag
+      Added Support for User-Facing Version
+      Added Support for Testing EG against Multiple Kubernetes Versions
+
+  - area: conformance
+    change: |
+      Enabled GatewayClassObservedGenerationBump conformance test
```
Comment (danehans) [inline release-notes/v0.3.0.yaml:47]: Remove since the test was disabled today due to flakes.

Comment (arkodg) [reply release-notes/v0.3.0.yaml:47]: this one wasn't @danehans

Comment (arkodg) [reply release-notes/v0.3.0.yaml:47]: the flakes were for `HTTPRouteObservedGenerationBump` and `GatewayObservedGenerationBump`

**Review by danehans — APPROVED** [review]

**Pull request merged**